install.sh

```diff
@@ -560,10 +576,16 @@ download_rancher_file() {
   local=$4
   local_location=$5
   source=$6
+  transactional=$7
+
+  TRANSACTIONAL_CMD=
+  if [ "${transactional}" = "true" ]; then
+      TRANSACTIONAL_CMD="transactional-update -d --no-selfupdate --continue --quiet run "
```

Comment:
There should be a way to override the`TRANSACTIONAL_CMD`, if a user needs to pass in specific or custom arguments to it.

Reply:
Furthermore, instead of calling it `TRANSACTIONAL_CMD`, we should probably rename this to `COMMAND_PREFIX` or something along those lines to allow for genericism. I do not look forward to having to rip out this environment variable later on, if we have to.

Reply:
Would you want it to be `COMMAND_PREFIX` or would `TRANSACTIONAL_COMMAND_PREFIX` work? Just thinking of longer term differentiation.